### ISSUE TITLE
Where to put attribution?

### ISSUE THREAD

**Issue by krzysztofpniak** [reactjs/reactjs.org#1787]
I'm working on my own library and I would love to have a good and user friendly docs like ReactJS has.
As I understand I can just use some of this repo parts under CC BY 4.0 license. I'm not sure how I should mention that my documentation is based on Reacts one. Any advices are welcome. I just want to do it in a fair way. Last question. Do I have to put my documentation into separate repository than my project is located? My library is under MIT license.

**Comment by rostislav-simonik** [comment]
Yes, I have the same question, I like the initial setup on gatsby which has been made, I wanna put my own content, own images, but wanna uses high part of src folder. So not sure where and what should be kept. If src folder would be under MIT, then everything would be clear.

**Comment by rostislav-simonik** [comment]
Maybe @bvaughn, you could help us to answer our question, If I can borrow a little of your time.  I've noticed that you have renamed license file. Many thanks.

**Comment by alexkrolick** [comment]
You might want to check out the Docusaurus project: https://github.com/facebook/docusaurus. It's not Gatsby but it generates static sites with a similar look and functionality to this site.

As far as the Gatsby source goes, you can probably include the original copyright notice in the license file for your derived works.

https://softwareengineering.stackexchange.com/questions/195076/mit-licence-copyright-notice-for-derived-works

**Comment by bvaughn** [comment]
(IANAL) I think the source code for this site is MIT and the content is CC-by-4.

**Comment by rostislav-simonik** [comment]
@bvaughn Thanks, (IANAL) Actually this can be tricky, because if its content is CC-by-4, and the rest doesn't contain specific  license declaration, then by default it's i don't know, unlicensed?. So that's my concern. I don't want to violate any license rights.

**Comment by samajammin** [comment]
Same question. I'm not interested in re-using any of the content, just the code for the gatsby app. @krzysztofpniak @rostislav-simonik did you resolve this? What approach did you take?

As @krzysztofpniak mentioned, I see [one file](https://github.com/reactjs/reactjs.org/blob/master/src/templates/components/Sidebar/ScrollSyncSection.js) with the header:

```
/**
 * Copyright (c) 2013-present, Facebook, Inc.
 *
 * This source code is licensed under the CC-BY-4.0 license found
 * in the LICENSE file in the root directory of this source tree.
 *
 * @emails react-core
 */
```

And [another file](https://github.com/reactjs/reactjs.org/blob/master/src/utils/loadScript.js) with the header:

```
/**
 * Copyright (c) 2013-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @emails react-core
 */
```

But the majority now just say:
```
/**
 * Copyright (c) 2013-present, Facebook, Inc.
 *
 * @emails react-core
 */
```

Thanks in advance!

**Comment by rostislav-simonik** [comment]
@samajammin we haven't used that code.

**Comment by samajammin** [comment]
@gaearon - sorry to bug you but perhaps you know the answer to this? Can you confirm bvaughn's comment that the source code for this site is MIT and the content is CC-by-4?
